### ISSUE TITLE
NTR: Prenatal ultrasound: abdomen small (Lab: ultrasound & echocardiogram)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8684]
New term request Prenatal ultrasound: abdomen small (elthat_101002140217):

**Comment by pnrobinson** [comment]
@MickeySegal
are we referring to Small fetal abdominal circumference?
https://www.ajog.org/article/S0002-9378(16)31040-7/fulltext

**Comment by MickeySegal** [comment]
Yes.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000339
- label: Small fetal abdominal circumference
- parents: [Abnormal fetal gastrointestinal system morphology (HP:0034207)](https://hpo.jax.org/app/browse/term/HP:0034207)
- pmid: PMID:30646865
- def: The distance around the abdomen of a fetus, generally as measured by prenatal sonography, is below the lower limit of normal.
- comment: This finding may be associated with fetal growth restriction.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8684
- synonyms:
